compose.yaml

```
@@ -21,9 +21,10 @@ services:
     depends_on:
       - db
       - frontend
+
   db:
     container_name: "db"
-    image: postgres:12.3-alpine
+    image: postgres:14.1
```

It's better to only specify the major version so we always use the latest available minor/patch version for that major.

```suggestion
    image: postgres:14
```